suite/tests/common/decode.c

```diff
@@ -708,9 +708,15 @@ DECL_EXTERN(func_ptr)
 #define FUNCNAME test_rip_rel_ind
         DECLARE_FUNC_SEH(FUNCNAME)
 GLOBAL_LABEL(FUNCNAME:)
+        sub      REG_XSP, FRAME_ALIGNMENT - ARG_SZ  /* Maintain alignment. Some systemn's
```

Comment:
This makes an illegal Windows prologue without an SEH64 directive.  Probably we should have an asm_defines macro for this to make it easy.  The directive would be `.allocstack <size>` I believe.

Reply:
Does '.allocstack' add to the instruction for the Windows compiler, or it's instead of the instruction? There's barely any allocstack directive in the code base right now, how come?
git grep allocstack
core/arch/asm_defines.asm:#  define PUSH_NONCALLEE_SEH(reg) push reg @N@ .allocstack 8

Reply:
It augments: it is not an instrution on its own.  It's not needed in leaf functions which do not touch the stack pointer so most of our asm code doesn't need it.  Most of our prologues just use PUSH_SEH.

Reply:
Ok. PTAL.